src/schema/nmdc.yaml

```diff
@@ -2643,6 +2644,16 @@ slots:
   vendor:
     range: InstrumentVendorEnum
 
+  has_data_generation_parts:
```

Comment:
use has_process_parts

has_process_parts will be updated to not have a domain 
has_process_parts slot will be not required, ProtocolExecution will update its slot usage to say has_process_part is required